lib/spack/spack/main.py

```diff
@@ -363,7 +363,15 @@ def make_argument_parser(**kwargs):
         metavar='DIR', help="add a custom configuration scope")
     parser.add_argument(
         '-d', '--debug', action='store_true',
-        help="write out debug logs during compile")
+        help="write out basic debug messages")
+    parser.add_argument(
+        '-dd', action='store_true',
```

Comment:
Is there not an option in argparse to count the number of appearances of an argument?

Reply:
There is.  There should be one `-d` / `--debug` argument with `action="count"`.  See docs on `count` here:

https://docs.python.org/3/library/argparse.html#action

Reply:
Cool!  I missed that.

Just confirmed in `2.7.18`.  Will change.

Reply:
Changed.